Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from swiplserver import *\r\n\r\nwith PrologMQI() as mqi:\r\n\twith mqi.create_thread() as prolog_thread:\r\n\t\tresult = prolog_thread.query(\"set_prolog_flag(encoding,utf8).\")\r\n\t\tprint(result)\r\n\t\tresult = prolog_thread.query(\"consult(\\\"knowledge_base.pl\\\").\")\r\n\t\tprint(result)\r\n\t\tresult = prolog_thread.query(\"create_semester_schedule(\\\"Ingegneria Informatica\\\", 1, 1, X)\")\r\n\t\tprint(result[1])
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision 83b5993f2c86d2055ac36200db5d122e11de2d42)
+++ b/main.py	(date 1675355507081)
@@ -1,6 +1,6 @@
 from swiplserver import *
 
-with PrologMQI() as mqi:
+with PrologMQI(prolog_path = "c:/program files/swipl/bin") as mqi:
 	with mqi.create_thread() as prolog_thread:
 		result = prolog_thread.query("set_prolog_flag(encoding,utf8).")
 		print(result)
